src/components/TradeWidget/TokenRow.tsx

```diff
@@ -48,22 +59,85 @@ const WalletDetail = styled.div`
   }
 `
 
+function renderOptionLabel(token: TokenDetails): React.ReactNode {
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
```

Comment:
Shouldn't we avoid to inline styles like this?

Reply:
Yeah I did not have a good feeling about this. I'll refactor the styles in a new PR.

Reply:
Will address on #147